test/unit/playfestController.spec.js

```diff
@@ -1,26 +1,38 @@
 describe('playfestController', function(){
   beforeEach(module('playfestApp'));
 
-  var testCtrl;
-  var scope;
+  var testCtrl, scope, deferred;
 
-  beforeEach(inject(function($controller, $rootScope){
+  var festivalNames = ["Glastonbury", "Lovebox"]
+
+  var service = jasmine.createSpyObj('TalkToBackendService', ['getFestivalMatches']);
+
+  beforeEach(inject(function($controller, $rootScope, $q){
     scope = $rootScope.$new;
+    deferred = $q.defer();
```

Comment:
we should be able to remove this line since we're not using this promise (we declare it inline next line using $q.when() which is apparently a better idea